ddp-workspace/projects/ddp-sdk/src/lib/services/activity/activityQuestionConverter.service.ts

```diff
@@ -88,6 +88,10 @@ export class ActivityQuestionConverter {
                     questionBlock.setAnswer(answer, false);
                 } else {
                     questionBlock.answer = valueForQuestion;
+                    if (questionJson.questionType === 'TEXT' && questionJson.confirmEntry) {
+                        const textQuestion = questionBlock as ActivityTextQuestionBlock;
+                        textQuestion.confirmationAnswer = textQuestion.answer;
```

Comment:
Ran into an issue where the `ActivityMatchValidationRule` will fail whenever I reload the activity page or when we're in read-only mode. I realized that the rule is checking whether `confirmAnswer == answer` but we're not setting `confirmAnswer` anywhere when initializing. I think the json converter here is a good place to do that.

![rgp_bug_confirm](https://user-images.githubusercontent.com/5713833/116942446-99e87e80-ac3f-11eb-888d-01c5e800cb62.png)